strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/testing/repository/NpmTestRepository.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.testing.repository;
+
+import org.carlspring.strongbox.artifact.coordinates.NpmArtifactCoordinates;
+import org.carlspring.strongbox.providers.layout.NpmLayoutProvider;
+import org.carlspring.strongbox.storage.repository.RepositoryPolicyEnum;
+import org.carlspring.strongbox.testing.storage.repository.RepositorySetup;
+import org.carlspring.strongbox.testing.storage.repository.TestRepository;
+import org.springframework.core.annotation.AliasFor;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Documented
+@Retention(RUNTIME)
+@Target({PARAMETER, ANNOTATION_TYPE})
+@TestRepository(layout = NpmArtifactCoordinates.LAYOUT_NAME)
+public @interface NpmTestRepository
+{
+
+    @AliasFor(annotation = TestRepository.class)
+    String storageId() default "storage0";
```

Comment:
I think the default for this should be `storage-npm`.

Reply:
Updated